frontend/src/Track.elm

```diff
@@ -22,64 +24,54 @@ import PagesComponents.Organization_.Project_.Models.FindPathResult exposing (Fi
 -- all tracking events should be declared here to have a good overview of them
 
 
-openAppCta : String -> TrackEvent
-openAppCta source =
-    { name = "open-app-cta", details = [ ( "source", source ) ], enabled = True }
+openSharing : Maybe { e | project : { p | organization : Maybe { o | id : OrganizationId }, id : ProjectId } } -> TrackEvent
+openSharing erd =
+    createEvent "open-sharing" [] (erd |> Maybe.map .project)
```

Comment:
As now we will have a lot of events, what naming convention do we follow?
Also, let's review them to keep only useful ones or add important missing ones...